libs/translations-finder/src/lib/translations-finder.util.ts

```diff
@@ -0,0 +1,243 @@
+import * as glob from 'glob';
+import * as fs from 'fs';
+
+/**
+ * @description
+ * Extracts the string key of the translation from the text
+ *
+ * @method extractString
+ *
+ * @param {string} translation
+ * The whole text matched by regex
+ *
+ * @param {RegExp} translationKeyRegex
+ * The regex to extract the translation key from the matched text
+ *
+ * @return {string}
+ * Extracted string
+ */
+const extractString = (translation: string, translationKeyRegex: RegExp): string => {
```

Comment:
don't think this helper function is needed, in any case, the arguements for such function could have generic names, str: string, regex: RegExp

Reply:
Then I would have this:
![Screenshot from 2019-12-16 16-38-29](https://user-images.githubusercontent.com/49659466/70920244-87f24580-2022-11ea-9fa7-595cd42fc8e2.png)

Instead of this:
![Screenshot from 2019-12-16 16-39-19](https://user-images.githubusercontent.com/49659466/70920290-9f313300-2022-11ea-9e89-53e25e570077.png)

It depends what is more readable. I use it only here. Should I remove this function?